docs/releasing/publishing.md

```diff
@@ -3,15 +3,22 @@
 1. At stand up, person leading the release to tell the GOV.UK Design System team we are close to releasing so we can coordinate the final cutoff date. Once the cutoff date passes, do not add any further major changes to the release. We can still add small fixes before step 7 as long as we notify the content designer and technical writer. However, we should try to avoid adding too many fixes in this way, as it requires us to repeat some of steps 4-6.
 
 2. Developers to raise new issues in the team GitHub repositories ([govuk-frontend](https://github.com/alphagov/govuk-frontend), [govuk-frontend-docs](https://github.com/alphagov/govuk-frontend-docs), [govuk-prototype-kit](https://github.com/alphagov/govuk-prototype-kit)) to:
```

Comment:
```suggestion
2. Developers to raise new cards in the team GitHub repositories ([govuk-frontend](https://github.com/alphagov/govuk-frontend), [govuk-frontend-docs](https://github.com/alphagov/govuk-frontend-docs), [govuk-prototype-kit](https://github.com/alphagov/govuk-prototype-kit)) to:
```

Reply:
The Github terminology for what we create is a 'new issue' - do we still want to switch this to 'cards'?

Reply:
Hmmm yeah, possibly not...

I suggested switching because it seems like this doc uses 'issue' and 'card' interchangeably, and 'card' looked like the more common term.

To be consistent, should I replace all instances of 'card' with 'issue'?

Reply:
Potentially. I think we do use them interchangeably, probably because different tools have different terms for them, e.g: Github uses 'issues', Trello uses 'cards', JIRA uses 'tickets'. Github 'issues' does feel a bit more geared towards someone raising a bug report rather than internal pieces of work, but it's the tool we use so perhaps we should switch to that. The mixed use in this doc might also be due to us updating these docs before, during and after the teams switch from Trello to Github.

Reply:
Ah yep, that all makes sense, ta for the rationale. I'll add a commit to make our terminology consistent.

Reply:
@vanitabarrett I'll also update our [publish-from-a-support-branch doc](https://github.com/alphagov/govuk-frontend/blob/main/docs/releasing/publishing-from-a-support-branch.md) to have consistent terminology.

Reply:
Ah interesting, never thought of that, thanks @querkmachine!